app/workers/delete_orphan_withdrawal_reason_records_worker.rb

```diff
@@ -0,0 +1,9 @@
+class DeleteOrphanWithdrawalReasonRecordsWorker
```

Comment:
```suggestion
class DeleteDraftWithdrawalReasonRecordsWorker
```
The word orphan isn't sitting right